flyte_tests_manifest.json

```diff
@@ -1,154 +1,209 @@
-[{
+[
+  {
     "name": "core",
     "priority": "P0",
     "path": "core",
+    "examples": [
+      ["advanced_composition.chain_entities.chain_workflows_wf", {}],
+      ["advanced_composition.conditions.consume_outputs", { "my_input": 10.0 }],
+      ["advanced_composition.decorating_tasks.wf", { "x": 10 }],
+      ["advanced_composition.decorating_workflows.wf", { "x": 19.8 }],
+      ["advanced_composition.dynamics.wf", { "s1": "Pear", "s2": "Earth" }],
+      [
+        "advanced_composition.map_task.my_map_workflow",
+        { "a": [1, 2, 3, 4, 5] }
+      ],
+      [
+        "advanced_composition.waiting_for_external_inputs.sleep_wf",
+        { "num": 5 }
+      ],
+      ["basics.documenting_workflows.sphinx_docstring_wf", {}],
+      ["basics.hello_world.hello_world_wf", {}],
+      ["basics.named_outputs.simple_wf_with_named_outputs", {}],
+      ["basics.shell_task.shell_task_wf", {}],
+      ["basics.workflow.simple_wf", { "x": [1, 2, 3], "y": [1, 2, 3] }],
+      ["data_types_and_io.dataclass.dataclass_wf", { "x": 10, "y": 20 }],
+      ["data_types_and_io.enum_type.coffee_maker", { "coffee": "latte" }],
+      [
+        "data_types_and_io.file.normalize_csv_file",
+        {
+          "csv_url": "https://people.sc.fsu.edu/~jburkardt/data/csv/biostats.csv",
+          "column_names": [
+            "Name",
+            "Sex",
+            "Age",
+            "Heights (in)",
+            "Weight (lbs)"
+          ],
+          "columns_to_normalize": ["Age"]
+        }
+      ],
+      [
+        "data_types_and_io.folder.download_and_normalize_csv_files",
+        {
+          "csv_urls": [
+            "https://people.sc.fsu.edu/~jburkardt/data/csv/biostats.csv",
+            "https://people.sc.fsu.edu/~jburkardt/data/csv/faithful.csv"
+          ],
+          "columns_metadata": [
+            ["Name", "Sex", "Age", "Heights (in)", "Weight (lbs)"],
+            ["Index", "Eruption length (mins)", "Eruption wait (mins)"]
+          ],
+          "columns_to_normalize_metadata": [["Age"], ["Eruption length (mins)"]]
+        }
+      ],
+      ["data_types_and_io.structured_dataset.simple_sd_wf", { "a": 42 }]
+    ],
     "exitCondition": {
       "exit_success": true,
       "exit_message": ""
     }
-  },{
+  },
+  {
     "name": "integrations-hive",
     "priority": "P1",
     "path": "examples/hive_plugin",
     "exitCondition": {
       "exit_success": true,
-        "exit_message": ""
+      "exit_message": ""
     }
-  },{
+  },
+  {
     "name": "integrations-k8s-spark",
     "priority": "P1",
     "path": "examples/k8s_spark_plugin",
+    "examples": [
+      [
+        "k8s_spark_plugin.pyspark_pi.my_spark",
+        { "triggered_date": "2023-11-21T18:58:01" }
```

Comment:
`triggered_date` is supposed to be a datetime when we invoke the `k8s_spark_plugin.pyspark_pi.my_spark`. We should remove this argument and let the default value kick in.